Cassette/CASFileObjectQueue.m

```diff
@@ -75,4 +75,25 @@ - (void)clear {
     [self.queueFile clear];
 }
 
+#pragma mark - Helper Method
+
+- (id)unarchiveData:(NSData *)data {
+    id result;
+
+    if (@available(iOS 11.0, macOS 10.13, *)) {
+        NSError *error;
+        NSKeyedUnarchiver *unarchiver = [[NSKeyedUnarchiver alloc]
+                                         initForReadingFromData:data
+                                         error:&error];
+        [unarchiver setRequiresSecureCoding:NO];
+        result = [unarchiver decodeObjectForKey:NSKeyedArchiveRootObjectKey];
+        if (error != nil) {
+            NSAssert(NO, @"error unarchiving data: %@", error.localizedDescription);
```

Comment:
Will this crash the app?

Reply:
the way NSAsserts work is that in debug mode it will stop execution, but in release mode it does not

Reply:
ok i changed it to use `CASLOG`